Source/Scene/Model.js

```diff
@@ -512,11 +512,15 @@ define([
         this.colorBlendAmount = defaultValue(options.colorBlendAmount, 0.5);
 
         /**
-         * The {@link ClippingPlaneCollection} used to selectively disable rendering the model. Clipping planes are not currently supported in Internet Explorer.
+         * The {ClippingPlaneCollection} used to selectively disable rendering the model. Clipping planes are not currently supported in Internet Explorer.
+         * @private
          *
          * @type {ClippingPlaneCollection}
          */
-        this.clippingPlanes = options.clippingPlanes;
+        this._clippingPlanes = undefined;
+
+        var clippingPlanes = options.clippingPlanes;
+        ClippingPlaneCollection.setOwnership(clippingPlanes, this, '_clippingPlanes');
```

Comment:
Instead of these two lines, you could just call the setter directly.

`this.clippingPlanes = options.clippingPlanes`